### DISCUSSION TITLE
Free tier 2 projects for students very limited

### DISCUSSION THREAD

**Discussion by jaydonteh** [supabase/supabase#4994] [Feature Requests]
Hi there, I'm an educator teaching kids coding in schools and recently I found the limits of 2 projects per account. I wish we've a little more projects to spare. At the moment my students would just get a second or third email account for this which I think is just a tad bit redundant if only the limit is higher.

I suggest perhaps to put inactive projects to a pause state should suffice, limiting the amount of concurrent active projects available for the free tiers. What does the community think about this? Would love to hear from you guys 🙂

**Comment by Roni1993** [comment]
i think while it's not 100% "clean design" you could easily just add a "task1_" in front of the table names. This would allow you to create tables & entries as long as you have space left.  
Another option also might be to add new schemas within the Database. I saw that there seems to be at least some support for it.
https://supabase.com/blog/2021/11/05/supabase-beta-october-2021
 
 I think the 2 free projects that are provided are already more than many other providers allow

**Reply by davidmaillo** [reply]
This is the solution that I am adapting, the best I have found so far. However I am concerned about mixing the Auth and Settings of several projects. I would like Supabase to expand the limits so that its proposal continues to make sense: "Be able to test and create before paying for the DB"

**Comment by Alm0stEthical** [comment]
any update ^?
